app/views/play/level/LevelLoadingView.coco.coffee

```diff
@@ -262,8 +262,13 @@ module.exports = class LevelLoadingView extends CocoView
     @$el.find('.license-required').show()
 
   onLoadError: (resource) ->
+    startCase = (str) -> str.charAt(0).toUpperCase() + str.slice(1)
     @$el.find('.level-loading-goals, .tip, .progress-or-start-container').hide()
-    @$el.find('.could-not-load').show()
+    if resource.resource.jqxhr.status is 404
+      @$el.find('.resource-not-found>span').text($.i18n.t('loading_error.resource_not_found', {resource: startCase(resource.resource.name)}))
+      @$elfind('.resource-not-found').show()
```

Comment:
lol, are you sure it works? 😛  Seems like it doesn't make a difference because default stage is visible anyway for this div!!

Reply:
oh but i missed a `.`